### DISCUSSION TITLE
Make video fill screen?

### DISCUSSION THREAD

**Discussion by elipsoid-cz** [iina/iina#4313] [Q&A]
Hi, I'm looking for a way to fill my screen with video, ie. remove all pillar- and letterboxing, ideally using one key. Is such thing possible?

In VLC this can be done by selecting a specific crop ratio, but it is cumbersome (I have to cycle through all of them every time I want to do this). 

iina seems to be missing my widescreen's aspect ratio in the crop menu and I don't really understand how to set it precisely using the custom crop option.

Thanks!

**Comment by paulodanilolima** [comment]
Maybe, you can try using AVPlayer in AVKIT, the commands like play or pause are there by default, you only have to actived them using showPlaybackControls
